### DISCUSSION TITLE
export to twitter account

### DISCUSSION THREAD

**Discussion by maro7777** [jonjomckay/fritter#416] [Q&A]
Hey, ist it possible to export my subscription from Fitter to an regular twitter account? I would like to have the option to write and reply to twittes again.

**Comment by ghost** [comment]
This wouldn't ever be possible as twitter doesn't have subscription importing, as I am aware of

**Comment by maro7777** [comment]
Oh man, i just opened a regular twitter account and i am missing fritter so much. Twitter is so diluted. Fitter was so good. 


On June 12, 2023 7:15:05 PM GMT+02:00, Harley ***@***.***> wrote:
>This wouldn't ever be possible as twitter doesn't have subscription importing, as I am aware of
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/jonjomckay/fritter/discussions/416#discussioncomment-6154925
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
